PantryPackage/PantryPackage/R/infofun.R

```diff
@@ -0,0 +1,110 @@
+#' Gives full recipe information with input of ingredients and dietary restriction
+#'
+#' @param ingredients A character vector of individual ingredients;
```

Comment:
Honestly I think everything looks good here, nothing to comment on :)

Reply:
Not sure if I just didn't know where to look but I could not find the vignette / tutorial?

Reply:
> Not sure if I just didn't know where to look but I could not find the vignette / tutorial?

it's in the commit "working on the shiny app and markdown".